.deployment/barista/Jenkinsfile

```diff
@@ -46,11 +46,25 @@ pipeline {
           }
         }
 
-        stage("Add internal Icons") {
+        stage("Internal Preparations") {
           when {
             expression { return params.INTERNAL_VERSION }
           }
+          environment {
+            BARISTA_SEARCH_ENDPOINT = credentials('barista-search-endpoint')
+            ENVIRONMENT_FILE = 'apps/barista-design-system/src/environments/environment.internal.ts'
+          }
           steps {
+
+            dir(BARISTA_PATH) {
+              script {
+                def text = readFile(file: "${ENVIRONMENT_FILE}")
+                text = text.replaceAll("\\{\\{BARISTA_INTERNAL_SEARCH_ENDPOINT\\}\\}", "${BARISTA_SEARCH_ENDPOINT}")
+                echo text
```

Comment:
Do we want to echo this or was this just for testing purposes?

Reply:
we want to echo this, we had this in the previous deployment as well to see immediately if the environment replace has worked or if an error occurred. These logs are protected and only accessible from the internal network.